scripts/inc.writeGlobalConfig.sh

```diff
@@ -332,6 +342,7 @@ CMDSEEKBACK=`grep 'CMDSEEKBACK' $PATHDATA/../settings/rfid_trigger_play.conf|tai
 # SECONDSWIPE
 # SECONDSWIPEPAUSE
 # SECONDSWIPEPAUSECONTROLS
+# MODELEGACY
```

Comment:
I would suggest to name this a bit more specific. Right now it pretty general to be only used in one case ^^

Something like `RFIDREADERRC522READMODELEGACY` and filename `Rfidreader_Rc522_Readmode_Legacy` ?
I know a bit clunky, what you know what you get ;)

Reply:
My intent was to give version 2 a legacy mode which can be used for other cases as well. Whether or not this is reasonable may be judged by you or others.

If there should be no general legacy mode, I would prefer to add the legacy-flag for the rc522 in the `deviceName.txt` to have all at the same place, e.g. as second value behind the `;` which is currently used as split parameter. In that case I would go with your suggestion of "fulluid" as keyword.
However, I think it does make sense to get implementations in version 2 similar to version 3 when doing the same stuff.

I am happy to receive votes or comments regarding the direction to go :-)
:a:  general legacy mode
:b: minimal solution for legacy option of rc522

Reply:
I vote for 🅱️ and only implement it for the rc522.
I don't see a use for a general legacy mode, as it would be to generic too. It only makes sense for single parts of some components if needed.

Changing this to `deviceName.txt` would have to much impact in my opinion. Also thats not what the file is about. Such configuration is generally handled by this global.conf files (compare other components), so its fine to use it.

Reply:
All right. Has been changed, cf. d618ec0db13a052664effedcf585677624a13bb5.